install-php-extensions

```diff
@@ -1377,6 +1383,27 @@ installPECLModule() {
 	esac
 }
 
+# Installs pickle to use instead of pecl
+# and add the zip extension that it requires, making sure its installed first
+installPickle() {
+	for PHP_MODULE_TO_INSTALL in $PHP_MODULES_TO_INSTALL; do
+		if ! stringInList "$PHP_MODULE_TO_INSTALL" "$BUNDLED_MODULES"; then
+			if test $PHP_MAJMIN_VERSION -lt 800; then
+				pecl channel-update pecl.php.net || true
+				PECL_COMMAND="pecl install"
+				return
+			fi
+			curl -L -f https://github.com/FriendsOfPHP/pickle/releases/latest/download/pickle.phar -o /tmp/pickle
+			chmod +x /tmp/pickle
+			if ! stringInList 'zip' "$(getPHPInstalledModules)"; then
+				PHP_MODULES_TO_INSTALL="zip $(removeStringFromList 'zip' "$PHP_MODULES_TO_INSTALL")"
```

Comment:
Do we need to remove `zip` if it's not included?

Reply:
Sorry, your code is correct.

Reply:
if your asking - i wondered about trying to remove zip in clean up if the user didnt ask for it , but i think that would be quite complicated to add to the code as there is no support for removing extensions at the moment.